src/net_builtin.c

```diff
@@ -501,6 +510,8 @@ static size_t tx_tcp(struct mg_tcpip_if *ifp, uint8_t *dst_mac, uint32_t dst_ip,
   tcp->flags = flags;
   tcp->win = mg_htons(MIP_TCP_WIN);
   tcp->off = (uint8_t) (sizeof(*tcp) / 4 << 4);
+  // if (flags & TH_SYN) tcp->off = 0x70;  // Handshake? header size 28 bytes
+
```

Comment:
Perhaps this can be conditionally enabled ? (follows from other case)
Being in two different places makes it easy to overlook. I suggest we either remove these or leave them with a conditional, whether we use it or not.